File: app/src/main/java/mattframpton/endclothing_mattframpton/network/EndClothingService.java
package mattframpton.endclothing_mattframpton.network;

import io.reactivex.Single;
import mattframpton.endclothing_mattframpton.model.ProductList;
import retrofit2.http.GET;

public interface EndClothingService {
	@GET("/media/catalog/example.json")
	Single<ProductList> getJSON();
}

File: app/src/main/java/mattframpton/endclothing_mattframpton/network/EndClothingService.kt
package mattframpton.endclothing_mattframpton.network

import io.reactivex.Single
import mattframpton.endclothing_mattframpton.model.ProductList
import retrofit2.http.GET

interface EndClothingService {
    @get:GET("/media/catalog/example.json")
    val json: Single<ProductList>
}
